File: pinyougou_sellergoods_interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.entity.PageResult;
import com.pinyougou.pojo.TbBrand;

import java.util.List;

public interface BrandService {
    public List<TbBrand> findAll();

    PageResult findPage(Integer pageNum, Integer pageSize);

    void add(TbBrand brand);

    TbBrand findOne(Long id);

    void update(TbBrand brand);

    void deleteBrand(Long[] ids);
}

File: pinyougou_manager_web/src/main/webapp/js/service/brandService.js
mt.service("brandService", function ($http) {
    this.findAll= function () {
        return $http.get("../brand/findAll.do");
    };

    this.findPage = function (pageNum, pageSize) {
        return $http.get("../brand/findPage.do?pageNum=" + pageNum + "&pageSize=" + pageSize);
    };

    this.update = function (entity) {
        return $http.post("../brand/update.do", entity);
    };

    this.add = function (entity) {
        return $http.post("../brand/add.do", entity);
    };

    this.findOne = function (id) {
        return $http.get("../brand/findOne.do?id=" + id);
    };

    this.dele = function (selectIds) {
        return $http.get("../brand/deleBrands.do?ids=" +selectIds);
    };
});